packages/app/package.json

```diff
@@ -6,19 +6,19 @@
     "private": true,
     "homepage": ".",
     "devDependencies": {
-        "@dhis2/cli-app-scripts": "^3.2.5",
+        "@dhis2/cli-app-scripts": "^4.0.7",
         "cypress": "^3.6.1",
         "eslint-plugin-cypress": "^2.7.0",
         "redux-mock-store": "^1.5.3"
     },
     "dependencies": {
-        "@dhis2/analytics": "4.3.25",
+        "@dhis2/analytics": "^4.4.1",
         "@dhis2/app-runtime": "*",
         "@dhis2/d2-i18n": "*",
-        "@dhis2/d2-ui-core": "^6.5.11",
-        "@dhis2/d2-ui-file-menu": "^6.5.11",
-        "@dhis2/d2-ui-interpretations": "^6.5.11",
-        "@dhis2/data-visualizer-plugin": "34.3.31",
+        "@dhis2/d2-ui-core": "^7.0.3",
+        "@dhis2/d2-ui-file-menu": "^7.0.3",
+        "@dhis2/d2-ui-interpretations": "^7.0.3",
+        "@dhis2/data-visualizer-plugin": "^34.3.32-alpha.1",
```

Comment:
why alpha?

Reply:
The new patch version hasn't been published yet.